CHTCollectionViewWaterfallLayout.m

```diff
@@ -6,6 +6,7 @@
 //
 
 #import "CHTCollectionViewWaterfallLayout.h"
+#import "tgmath.h"
```

Comment:
For 64bit support.
http://stackoverflow.com/questions/5352457/cgfloat-based-math-functions/5352779#5352779